.github/workflows/docker-image.yml

```
@@ -39,7 +39,91 @@ jobs:
+          DOCKER_BUILDKIT: 1
+        shell: bash
+        run: |
+          DOCKER_BUILDKIT=1 docker compose up -d
+
+      - name: Wait for init
+        shell: bash
+        run: while ! docker compose logs web --tail 10 | grep -q '>>>>> run-maintenance-script.sh <<<<<'; do sleep 1; done
+
+      - name: Wait for container to be healthy
+        shell: bash
+        run: while ! docker inspect --format='{{json .State.Health.Status}}' $(echo $(basename $(pwd)) | tr '[:upper:]' '[:lower:]')-web-1 | grep -q 'healthy'; do sleep 1; done
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2005:style:1:72: Useless echo? Instead of 'echo $(cmd)', just use 'cmd'

(shellcheck)